.gitlab/build_quartz.yml

```diff
@@ -28,7 +28,7 @@ quartz_allocate:
   script:
     # Use when ellastic ci is on quartz or we go back to ruby
     #- salloc --reservation=ci --qos=ci_ruby -N 1 -c 36 -t 60 --no-shell --job-name=${PROJECT_ALLOC_NAME}
-    - salloc -N 1 -c 36 -t 60 --no-shell --job-name=${PROJECT_ALLOC_NAME}
+    - salloc -N 1 -c 36 -t 60 -p pdebug --no-shell --job-name=${PROJECT_ALLOC_NAME}
```

Comment:
I looked at these jobs when I did the TOSS4 PR and I am not confident the `srun` 's are staying inside the outside `salloc`. It was allocating then sitting at the inner jobs waiting for more resources.

Reply:
> I looked at these jobs when I did the TOSS4 PR and I am not confident the `srun` 's are staying inside the outside `salloc`. It was allocating then sitting at the inner jobs waiting for more resources.

Think I saw the same thing.

`srun` failing to find existing allocation (no debug flag for `srun`):
https://lc.llnl.gov/gitlab/axom/axom/-/pipelines/290398

`srun` after adding debug flag:
https://lc.llnl.gov/gitlab/axom/axom/-/pipelines/290406